pennylane/math/matrix_manipulation.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains methods to expand the matrix representation of an operator
+to a higher hilbert space with re-ordered wires."""
+import pennylane as qml
```

Comment:
I wonder if we can import just what we need from `multi_dispatch`?

But namespacing and import order can be hard, so if it's not working easily, this is fine.

Reply:
We make use of `qml.wires.Wires` 😞, which is not imported in `multi_dispatch`. A lot of functions that are used in this method do come from the `~/math/` folder itself. We could change those imports from `qml.math._func_` to just `_func_` and from the top do `from ./folder/ import _func_`. But we will still need to at least import `Wires`

Reply:
I bet we could even get rid of the dependency on `Wires`.  But that's a problem for another day.